src/biocframe/BiocFrame.py

```diff
@@ -1019,6 +1029,68 @@ def remove_columns(self, columns: Sequence[Union[int, str]], in_place: bool = Fa
 
         return output
 
+    def remove_row(self, row: Union[int, str], in_place: bool = False) -> "BiocFrame":
+        """Remove a row. This is a convenience wrapper around :py:attr:`~remove_rows`.
+
+        Args:
+            row:
+                Name or positional index of the row to remove.
+
+            in_place:
+                Whether to modify the object in place. Defaults to False.
+
+        Returns:
+            A modified ``BiocFrame`` object, either as a copy of the original
+            or as a reference to the (in-place-modified) original.
+        """
+        return self.remove_rows([row], in_place=in_place)
```

Comment:
every though the typehints suggest `row` should be a scalar, people don't read the docs. I would check if row is a list or not and only then coerce to a list. something like this. 

```py
if not isintance(row, (int, str):
  row = list(row)
```